runtime/context.go

```diff
@@ -171,6 +173,29 @@ func GetScopeFieldsFromCtx(ctx context.Context) map[string]string {
 	return fields
 }
 
+// WithEndpointScopeFields returns a new context with the given scope fields attached to context.Context
```

Comment:
The function description is the same as `WithRequestScopeFields`, I can't tell what's the semantic difference between these two. What is the difference and why is it necessary?

Reply:
WithEndpointScopeFields sets tags at endpoint level to context, WithRequestScopeFields sets tags per request to context